coralproject/talk
# Allow fullscreen in external media

## What does this PR do?
This PR should solve the problem of embedded videos not being able to be fullscreen. It adds attributes to the iframe inside of `<Frame />` to ensure fullscreen is enabled.

## What changes to the GraphQL/Database Schema does this PR introduce?
None.

## Does this PR introduce any new environment variables or feature flags? 
No.

## If any indexes were added, were they added to `INDEXES.md`?
n/a

## How do I test this PR?
Start up coral, find a tweet with a video, or a youtube video, add it to a comment, and observe that you are able to switch the video to fullscreen after the comment is submitted.
